src/test/java/org/prebid/server/it/IxTest.java

```diff
@@ -33,4 +33,20 @@ public void openrtb2AuctionShouldRespondWithBidsFromIx() throws IOException, JSO
         // then
         assertJsonEquals("openrtb2/ix/test-auction-ix-response.json", response, singletonList("ix"));
     }
+
+    @Test
+    public void openrtb2AuctionShouldRespondWithBidsFromIxForPrivacySandboxRequest() throws IOException, JSONException {
+        // given
+        WIRE_MOCK_RULE.stubFor(post(urlPathEqualTo("/ix-exchange"))
+                .withRequestBody(equalToJson(jsonFrom("openrtb2/ix/privacy-sandbox-test-ix-bid-request.json",
+                        prebidVersionProvider)))
+                .willReturn(aResponse().withBody(jsonFrom("openrtb2/ix/privacy-sandbox-test-ix-bid-response.json"))));
+
+        // when
+        final Response response = responseFor("openrtb2/ix/privacy-sandbox-test-auction-ix-request.json",
+                Endpoint.openrtb2_auction);
+
+        // then
+        assertJsonEquals("openrtb2/ix/privacy-sandbox-test-auction-ix-response.json", response, singletonList("ix"));
+    }
```

Comment:
do not need to create a separate integration test and a separate set of the json files, just use your json files for the existing test

Reply:
Our integration tests mainly for the bidder configuration verification. But, since your test case extends the regular one, I suggest using yours payloads instead. So please replace existing JSON payloads with yours.

Reply:
Ok, updated the base case.